File: Scala/start-code/startcode.scala
// Beschreibung: Ein einfaches Scala-Programm
// Datum: 25.01.2025

object Main {
  def main(args: Array[String]): Unit = {
    // Begrüßung
    println("Willkommen zu meinem Scala-Programm!")

    // Funktion definieren
    def begruessung(): Unit = {
      println("Das ist eine Beispiel-Funktion in Scala.")
    }

    // Funktion aufrufen
    begruessung()

    // Nutzereingabe
    print("Bitte geben Sie Ihren Namen ein: ")
    val name = scala.io.StdIn.readLine()

    // Begrüßung mit dem Namen
    println(s"Hallo, $name! Schön, dass Sie hier sind.")

    // Programm beenden
    println("Das Programm wurde beendet.")
  }
}

File: Perl/start-code/startcode.pl
#!/usr/bin/perl
# Beschreibung: Ein einfaches Perl-Programm
# Datum: 25.01.2025

use strict;
use warnings;

# Begrüßung
print "Willkommen zu meinem Perl-Programm!\n";

# Funktion definieren
sub begruessung {
    print "Das ist eine Beispiel-Funktion in Perl.\n";
}

# Funktion aufrufen
begruessung();

# Nutzereingabe
print "Bitte geben Sie Ihren Namen ein: ";
my $name = <STDIN>;
chomp($name);  # Entfernt den Zeilenumbruch am Ende der Eingabe

# Begrüßung mit dem Namen
print "Hallo, $name! Schön, dass Sie hier sind.\n";

# Programm beenden
print "Das Programm wurde beendet.\n";
